package.json

```diff
@@ -35,9 +35,9 @@
   "readmeFilename": "README.md",
   "license": "BSD-2-Clause",
   "devDependencies": {
+    "@hapi/lab": "^24.5.1",
```

Comment:
Had to upgrade this because the espree used by 14.x didn't support the regex lookbehind.